script/model/forecast.py

```diff
@@ -0,0 +1,520 @@
+#!/usr/bin/env python3
+"""
+TODO(ricky):
+The modeling scripts should be under a project, with __init__
+What should be the best dependency between the modeling scripts and testing package?
```

Comment:
You're right that there should be a separation between the forecasting script and the testing script. Basically, the `DataLoader` and `LSTM` classes should be under `script/forecasting`. And everything else should be under `script/testing/forecasting`, where we invoke the test script for the Jenkins pipeline.